### PR TITLE
Added FTP Get and Put operations

### PR DESCRIPTION
**Pull request by harjocsergiu**
Added FTP Get and Put operations

### CODE REVIEW

File: cs-rft/src/main/java/io/cloudslang/content/rft/utils/ftp/FTPInputsValidation.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            @NotNull final String characterSet) {
+
+        final List<String> exceptionMessages = new ArrayList<>();
+
+        addVerifyNotNullOrEmpty(exceptionMessages, hostName, PARAM_HOSTNAME);
+        addVerifyPort(exceptionMessages, port);
+        addVerifyNotNullOrEmpty(exceptionMessages, localFile, PARAM_LOCAL_FILE);
+        addVerifyNotNullOrEmpty(exceptionMessages, remoteFile, PARAM_REMOTE_FILE);
+        addVerifyNotNullOrEmpty(exceptionMessages,user,PARAM_USER);
+        // addVerifyNotNullOrEmpty(exceptionMessages,password,PARAM_PASSWORD);
```
Comment (IuliaMoldovan) [inline cs-rft/src/main/java/io/cloudslang/content/rft/utils/ftp/FTPInputsValidation.java]: this should be commented ?

Comment (harjocsergiu) [reply cs-rft/src/main/java/io/cloudslang/content/rft/utils/ftp/FTPInputsValidation.java]: Changes were made

File: cs-rft/src/main/java/io/cloudslang/content/rft/actions/ftp/Get.java [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+                                       @Param(value = PARAM_PORT, description = PARAM_PORT_DESC) String port,
+                                       @Param(value = PARAM_LOCAL_FILE, description = PARAM_LOCAL_FILE_DESC) String localFile,
+                                       @Param(value = PARAM_REMOTE_FILE, description = PARAM_REMOTE_FILE_DESC) String remoteFile,
+                                       @Param(value = PARAM_USER, description = PARAM_USER_DESC) String user,
+                                       @Param(value = PARAM_PASSWORD, description = PARAM_PASSWORD_DESC) String password,
+                                       @Param(value = PARAM_TYPE, description = PARAM_TYPE_DESC) String type,
+                                       @Param(value = PARAM_PASSIVE, description = PARAM_PASSIVE_DESC) String passive,
+                                       @Param(value = PARAM_CHARACTER_SET, description = PARAM_CHARACTER_SET_DESC) String characterSet) {
+
+        hostName = defaultIfEmpty(hostName, EMPTY);
```
Comment (ericandreica) [inline cs-rft/src/main/java/io/cloudslang/content/rft/actions/ftp/Get.java]: For required inputs it's no need to assing them empty and verify after that if they are empty.

File: cs-rft/src/main/java/io/cloudslang/content/rft/services/FTPService.java [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+
+    public Map<String, String> ftpOperation(FTPInputs ftpInputs, FTPOperation ftpOperation) {
+        FTPClient ftp = null;
+        final Map<String, String> result = new HashMap<>();
+
+        try {
+            // open server portNumber with a specified command encoding
+            ftp = connect(ftpInputs.getHostname(), Integer.parseInt(ftpInputs.getPort()), ftpInputs.getCharacterSet(), Boolean.parseBoolean(ftpInputs.getPassive()));
+
+            // user foo
```
Comment (ericandreica) [inline cs-rft/src/main/java/io/cloudslang/content/rft/services/FTPService.java]: We need these comments?

Comment (harjocsergiu) [reply cs-rft/src/main/java/io/cloudslang/content/rft/services/FTPService.java]: done

File: cs-rft/src/main/java/io/cloudslang/content/rft/actions/ftp/Get.java (hunk 2 above)
Comment (harjocsergiu) [reply cs-rft/src/main/java/io/cloudslang/content/rft/actions/ftp/Get.java]: Fixed

**Pull request merged**